push_notifications/gcm.py

```diff
@@ -67,7 +66,7 @@ def _gcm_send_plain(registration_id, data, collapse_key=None, delay_while_idle=F
 		values["time_to_live"] = time_to_live
 
 	for k, v in data.items():
-		values["data.%s" % (k)] = v.encode("utf-8")
+		values["data.%s" % (k)] = str(v).encode("utf-8")
```

Comment:
because above I replaced some lines, here I just edited in the middle. I didn't want to get a huge diff just from these replacements...